docs/scripts/api/utils.ts

```diff
@@ -1,8 +1,8 @@
 import { Annotation } from 'doctrine';
 import kebabCase from 'lodash/kebabCase';
 import * as prettier from 'prettier';
-import * as fse from 'fs-extra';
 import { Symbol, isPropertySignature, isExportSpecifier, TypeFormatFlags } from 'typescript';
+import fs from 'node:fs';
```

Comment:
Perhaps we can standardize on

```suggestion
import fs from 'node:fs/promises';
```